src/components/CardView/__test__/__snapshots__/CardView.test.js.snap

```diff
@@ -6,10 +6,10 @@ exports[`renders correctly 1`] = `
 >
   <img
```

Comment:
You forgot to use the Image component here?

Reply:
this is just a snapshots bro

Reply:
the img in the component is next image but when the test work the snapshot is convert the code to be simple html so the image will be img